src/scss/_spark-theme.scss

```
@@ -49,4 +49,8 @@ body {
   visibility: hidden;
 }
 
+.docs-b-SkipNav {
```

```suggestion
.docs-c-SkipNav {
```
Would be "c" for component instead of "b" for base element.